RMSSD Function.R

```diff
@@ -0,0 +1,185 @@
+#RMSSD Function
+rm(list=ls())
+
+dataset <- `2019.7.10_RR_p36t0`
+
+#HR data may be outputed from equipment as RR values or ms/time values
+#let the individual specify what type of data they have
+
+data <- as.numeric(dataset[,2])
+x <- data.frame(data)
+
+
+
+
+#RMSSD FUNCTION
+#most common measure of heart rate variability 
+myRMSSD <- function(x, input, segment, max, min) {  
+
+  if (input == "RR" && segment == FALSE) {
+    N <- length(data)
+    N <- N-1
+    vector <- x^2
+    new <- sum(vector)/N
```

Comment:
try to name you objects as understandable as possible. 
What does the object "new" stand for? Is it a mean of some sort? 
The same goes for "vector".